src/com/google/foam/demos/u2/Border.js

```diff
@@ -20,9 +20,19 @@ foam.CLASS({
   name: 'Tab',
   extends: 'foam.u2.Element',
 
+  css: `
+    ^ { xxxposition: absolute; }
```

Comment:
This block doesn't do anything and could be removed. Actually all changes to demos should be removed from this PR.